lua/lazyvim/plugins/extras/lang/flutter.lua

```diff
@@ -0,0 +1,30 @@
+-- requires dart/flutter to be installed
+return {
+  recommended = {
+    ft = "dart",
+    root = { "pubspec.yaml" },
+  },
+  {
+    "akinsho/flutter-tools.nvim",
+    event = "VeryLazy",
```

Comment:
Should this really be loaded every time Neovim opens? Maybe better to load it on `ft` instead? Don't know anything about Dart, so not sure if there are any problems if you load on `ft` instead. Just a suggestion.

Reply:
let me see if ft works

Reply:
it works, i updated this on fsharp pr too